src/libraries/Common/tests/Tests/Interop/OSReleaseTests.cs

```diff
@@ -52,7 +52,17 @@ public void GetPrettyName_CannotRead_ReturnsNull()
         {
             string path = CreateTestFile();
             File.SetUnixFileMode(path, UnixFileMode.None);
-            Assert.ThrowsAny<Exception>(() => File.ReadAllText(path));
+
+            // If user have root permissions, kernel doesn't care about access priviliges,
+            // so there is no point in expecting System.Exception
+            if (!Environment.IsPrivilegedProcess)
+            {
+                Assert.ThrowsAny<Exception>(() => File.ReadAllText(path)); 
+            }
+            else
```

Comment:
I would probably make the test conditional and if needed add separate set of tests for privilege access. I would expect more tests that checks permissions.

But in general I think is is ok if just want to get unblocked to run tests as root.

Reply:
I wasn't certain whether I should modify the test itself or the core library functionality, so this solution was more of a quick fix.

I like your approach of creating a separate set of tests specifically for privileged users. Do you think it's worth implementing in this case? If so, I can start working on that solution tomorrow.

Reply:
perhaps @dotnet/area-system-io  can provide more guidance. I personally feel the quick fix + follow-up would be best.